js/widgets/forms/autogrow.js

```diff
@@ -72,8 +73,10 @@ define( [
 			}
 		},
 
-		_unbindAutogrow: function() {
-			this.element.removeClass( "ui-textinput-autogrow" );
+		_unbindAutogrow: function( removeClass ) {
+			if ( removeClass ) {
+				this._removeClass( "ui-textinput-autogrow" );
+			}
```

Comment:
This conditional is not needed its fine to remove it if it makes sense as part of other application logic.